src/reportWebVitals.js

```diff
@@ -1,6 +1,8 @@
-const reportWebVitals = onPerfEntry => {
+const reportWebVitals = (onPerfEntry) => {
   if (onPerfEntry && onPerfEntry instanceof Function) {
-    import('web-vitals').then(({ getCLS, getFID, getFCP, getLCP, getTTFB }) => {
+    import('web-vitals').then(({
+      getCLS, getFID, getFCP, getLCP, getTTFB,
+    }) => {
       getCLS(onPerfEntry);
       getFID(onPerfEntry);
       getFCP(onPerfEntry);
```

Comment:
- [ ] Thanks for making some the required changes. You still have multiple unneccessary files that are not related to the calculator app. e,g `logo192.png`, `logo152.png`, `favicon`, `reportWebVitals.js` e.t.c. Please remove any file that would not be directly of any use in the project.